app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class='footer'>
+  <small>
+    The <a href='hrrp://www.railstutorial.org/'>Ruby on Rails Tutorial</a>
+    by <a href='http://www.michaelharl.com/'>Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to 'About',    about_path %></li>
+      <li><%= link_to 'Contact',  contact_path %></li>
```

Comment:
I like the alignment between the two lines. However you've got an extra space here in the 'Contact' line that you don't need.